packages/neptune-extension/src/common/utils/projects.js

```diff
@@ -0,0 +1,9 @@
+import { backendClient } from '../api/backend-client';
+
+export async function fetchProjects() {
```

Comment:
Would you mind looking at `common/utils/checkout.ts`, it's already implemented there.

Reply:
Ok